### PR TITLE
Configure automatic deployment to Heroku

### PR DESCRIPTION
**Pull request by klochenok**
This PR describes steps of deploying app on Heroku by creating specific files and settings. This repo has been also activated for automatic deployment on Heroku.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,5 @@ db.sqlite3
 /static
 .DS_Store
 local_settings.py
+.python-version
+
```
Comment (webknjaz) [inline .gitignore]: This line is useless

File: mysite/settings.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 """
 
 import os
-
```
Comment (webknjaz) [inline mysite/settings.py:14]: I'd add a separator line here

File: mysite/settings.py [hunk 3]
```diff
@@ -78,8 +78,12 @@
 
 DATABASES = {
     'default': {
```
Comment (webknjaz) [inline mysite/settings.py]: Please find and fulfill my comment about dj_database_url

File: dev-requirements.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+-r requirements.txt
```
Comment (webknjaz) [inline dev-requirements.txt]: dev-only requirements must be moved here.

**Review by webknjaz — CHANGES_REQUESTED** [review]
Re-open the issue to do things right. Merging the PR should automatically close it.

File: mysite/settings.py (hunk 3 above)
Comment (klochenok) [reply mysite/settings.py]: As I understand you mean to put here `import dj_database_url`?

Comment (webknjaz) [reply mysite/settings.py]: No, the config retrieval

File: mysite/wsgi.py [hunk 5]
```diff
@@ -10,7 +10,10 @@
 import os
 
 from django.core.wsgi import get_wsgi_application
+from whitenoise.django import DjangoWhiteNoise
 
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'mysite.settings')
```
Comment (webknjaz) [inline mysite/wsgi.py:14]: According to doc whitenoise import should go next to this line

**Review by webknjaz — CHANGES_REQUESTED** [review]

File: .pre-commit-config.yaml [hunk 6]
```diff
@@ -2,8 +2,10 @@
   sha: master
   hooks:
   - id: trailing-whitespace
+    exclude: blog/migrations/
```
Comment (webknjaz) [inline .pre-commit-config.yaml]: Do you have any trailing whitespaces errors for this folder?

File: .pre-commit-config.yaml [hunk 7]
```diff
@@ -21,10 +23,10 @@
   - id: check-yaml
   - id: detect-private-key
 
-- repo: git://github.com/FalconSocial/pre-commit-mirrors-pep257
-  sha: master
-  hooks:
-  - id: pep257
+#- repo: git://github.com/FalconSocial/pre-commit-mirrors-pep257
```
Comment (webknjaz) [inline .pre-commit-config.yaml:25]: I think this is wrong

File: requirements/test.txt [hunk 8]
```diff
@@ -0,0 +1,2 @@
+-r ../requirements.txt
+pre-commit==0.9.4
```
Comment (webknjaz) [inline requirements/test.txt]: Upgrade to 0.10.1+

File: .pre-commit-config.yaml (hunk 6 above)
Comment (klochenok) [reply .pre-commit-config.yaml]: I have too long line error, it even doesn't get into "max-line-length" interval.

File: .pre-commit-config.yaml (hunk 7 above)
Comment (klochenok) [reply .pre-commit-config.yaml:25]: Agree, but I delayed fixing these errors. I had had this variant before we rebased branch, but this commit was lost.

**Review by pavlenk0 — APPROVED** [review]

File: .pre-commit-config.yaml (hunk 6 above)
Comment (webknjaz) [reply .pre-commit-config.yaml]: what do long lines have to do with trailing whitespaces hook?

File: mysite/wsgi.py [hunk 9]
```diff
@@ -8,9 +8,11 @@
 """
 
 import os
-
```
Comment (webknjaz) [inline mysite/wsgi.py:11]: Why did you remove the separation between stdlib packages and installed ones?

Comment (klochenok) [reply mysite/wsgi.py:11]: Fixed it

**Review by webknjaz — APPROVED** [review]

**Pull request merged**